src/components/Charts/utils.ts

```diff
@@ -196,3 +198,37 @@ export const zoneAnnotations = (
     labels: getZoneLabels(endDate, minYAxis, maxYAxis, value, zones),
   },
 ];
+
+export const getTruncationDate = (date: Date, truncationDays: number) =>
+  moment(date).subtract(truncationDays, 'days').toDate();
+
+export const randomizeId = (name: string): string =>
+  `${name}-${Math.random().toFixed(9)}`;
+
+export const getChartRegions = (minY: number, maxY: number, zones: Zones) => [
+  {
+    valueFrom: minY,
```

Comment:
could we just add this to the og files and not have this function?

Reply:
Not really, because `minY` and `maxY` depend on the data, they are calculated so all the regions are visible (even if the R(t) curve doesn't get to that region) and to crop the values for R(t) if they are too far from the medium and low zones, that way they are still legible in the chart. Makes sense?

Note: I still need to explain better region vs. zone, I will get to that soon!

Reply:
hmm seems like we oculd use negative infinitey and infinitey and then clip the charts, but if that doesn't work this is chill

Reply:
can't be done that way, the clipping area has to be bounded (although that's weird, the SVG surface is supposed to go to infinity in all directions)